beta-pipelines/skylab/m3c/CondensedSnm3C.wdl

```diff
@@ -87,20 +87,20 @@ workflow WDLized_snm3C {
           chrom_size_path = chromosome_sizes
    }
 
-   # call summary {
-   #     input:
-   #         trimmed_stats = Sort_and_trim_r1_and_r2.trim_stats,
-   #         hisat3n_stats = Hisat_3n_pair_end_mapping_dna_mode.hisat3n_stats,
-   #         r1_hisat3n_stats = hisat_single_end_r1_r2_mapping_dna_mode_and_merge_sort_split_reads_by_name.r1_hisat3n_stats,
-   #         r2_hisat3n_stats = hisat_single_end_r1_r2_mapping_dna_mode_and_merge_sort_split_reads_by_name.r2_hisat3n_stats,
-   #         dedup_stats = dedup_unique_bam_and_index_unique_bam.dedup_stats,
-   #         chromatin_contact_stats = call_chromatin_contacts.chromatin_contact_stats,
-   #         allc_uniq_reads_stats = unique_reads_allc.allc_uniq_reads_stats,
-   #         unique_reads_cgn_extraction_tbi = unique_reads_cgn_extraction.unique_reads_cgn_extraction_tbi
-   # }
+    call summary {
+        input:
+            trimmed_stats = Sort_and_trim_r1_and_r2.trim_stats_tar,
+            hisat3n_stats = Hisat_3n_pair_end_mapping_dna_mode.hisat3n_paired_end_stats_tar,
+            r1_hisat3n_stats = Hisat_single_end_r1_r2_mapping_dna_mode_and_merge_sort_split_reads_by_name.hisat3n_dna_split_reads_summary_R1_tar,
+            r2_hisat3n_stats = Hisat_single_end_r1_r2_mapping_dna_mode_and_merge_sort_split_reads_by_name.hisat3n_dna_split_reads_summary_R2_tar,
+            dedup_stats = dedup_unique_bam_and_index_unique_bam.dedup_stats_tar,
+            chromatin_contact_stats = call_chromatin_contacts.chromatin_contact_stats,
+            allc_uniq_reads_stats = unique_reads_allc.allc_uniq_reads_stats,
+            unique_reads_cgn_extraction_tbi = unique_reads_cgn_extraction.output_tbi_tar
+    }
 
     output {
-        #File MappingSummary = summary.MappingSummary
+        File MappingSummary = summary.mapping_summary
         #File allcFiles = unique_reads_allc.
```

Comment:
Should we add and/or clean-up these commented outputs. I think some of them may be included below